doc_admin/AdminManual.adoc

```diff
@@ -1931,22 +1949,18 @@ _log4j.properties_ as explained in the chapter <<log4j-properties,log4j.properti
 === 6.1 Update Open Hospital - CLIENT mode
 
 New releases, as well as this Admin Manual and the User Manual, are regularly released on Github/SourceForge. See the <<download,Download>> section for more details.
-
-There are two types of releases:
-
-. Minor Releases: only the third number changes (e.g. **1.8.**1 -> **1.8.**2)
-. Major Releases: the first or second number changes (e.g. **1.**7.3 -> **1.**8.0)
-
-*Minor Releases*:
-
-Changes are found in the CHANGELOG file.
-
 NOTE: Following instructions only apply between “official releases”.
 
 Minor releases (if not otherwise specified) do not need changes to the database structure, so the working copy can be replaced on each client with the new one and then connect to the same DB
 (see <<database-properties,database.properties>> and <<log4j-properties,log4j.properties>>).
 
 It is important to keep/preserve files that have been created or modified, especially configuration/settings and data files (see <<configuration,Configuration>>).
+=======
+Minor releases (if not otherwise specified) do not need changes to the database structure, so the working copy can be replaced on each client with the new one and then connect to the same DB
+(see <<database-properties,database.properties>> and <<log4j-properties,log4j.properties>>).
+
+It is important to keep/preserve files that have been created or modified, especially configuration/settings and data files (see <<config-files,Configuration Files>>).
+>>>>>>> develop
```

Comment:
this part looks like an unresolved conflict... 
the only difference I see is in the link `(see <<configuration,Configuration>>).` vs `(see <<config-files,Configuration Files>>).`

I think the former is the correct one, so the section between `=======` and `>>>>>>> develop` should be safe to remove

Reply:
Yes, correct is  "<<configuration,Configuration>>"